### ISSUE TITLE
Mouser price import faulty?

### ISSUE THREAD

**Issue by Tropaion** [Part-DB/Part-DB-server#540]
**Describe the bug**
I just tried the import function from mouser and created an Search-API for it.
Now I tried to update a current part, but the prices of the mouser part are wrong.
It also says that the part was announced even though it clearly is released and in stock

**To Reproduce**
Try importing this part to partdb: https://www.mouser.at/ProductDetail/Littelfuse/65600201009
It says announced:
![image](https://github.com/Part-DB/Part-DB-server/assets/6797663/add5b3bc-da96-4bc3-bfa4-807cf272dae5)
Prices all zero:
![image](https://github.com/Part-DB/Part-DB-server/assets/6797663/81131100-c2f4-49b3-9265-f5b335c0e310)
If the search already says anounced, shouldn't it not try to add prices?

**Expected behavior**

**Screenshots**

**Server Side**
- Part-DB Version: 1.11.0
- PHP Version: 8.1.27<html>
<body>

<html>
<body>

--

</body>
</html>
--

</body>
</html>
- Database Server [e.g. MySQL 5.7]

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by jbtronics** [comment]
The mouser api returns prices as locale dependent string depending on the selected country. Austrian mouser returned some string different to the expected one. This should be fixed with 1.11.1

Before, all parts which were marked as "New" by mouser were marked as "Announced". Now this only happens, if the part is new and no parts are instocked at mouser yet (otherwise its just active).

**Comment by Tropaion** [comment]
@jbtronics thanks, that was fast. Tried it just now, but accidently found another bug.
At some LCSC parts, it is importing the HTML-Tags:
![image](https://github.com/Part-DB/Part-DB-server/assets/6797663/44163c25-6bcf-4e6a-ae14-2891f3a8b182)

And I have two questions, is mass import from info provider possible/planned?
Is import of part paramters support? I just tried the same fuse holder and I don't get any parameters.
For example, you order 100 different parts from mouser you just import your order xml file.
